test/unit/communication/test_broadcast_panel.cpp

```diff
@@ -111,15 +111,23 @@ TYPED_TEST(PanelBcastTest, BroadcastRowStoreTransposed) {
       testBroadcast<TypeParam, Coord::Row, StoreTransposed::Yes>(cfg, comm_grid);
 }
 
-std::vector<config_t> test_params_bcast_transpose{
-    {{0, 0}, {1, 1}, {0, 0}},    // empty matrix
-    {{10, 10}, {2, 2}, {5, 5}},  // empty panel (due to offset)
-    {{20, 20}, {2, 2}, {9, 9}},  // just last tile (communicate without transpose)
-    {{25, 25}, {5, 5}, {1, 1}},
+struct ParamsBcastTranspose {
+  const GlobalElementSize sz;
+  const TileElementSize blocksz;
+  const GlobalTileIndex offset;
+  const GlobalTileIndex offsetT;
+};
+
+std::vector<ParamsBcastTranspose> test_params_bcast_transpose{
+    {{0, 0}, {1, 1}, {0, 0}, {0, 0}},      // empty matrix
+    {{10, 10}, {2, 2}, {5, 5}, {5, 5}},    // empty panel (due to offset)
+    {{20, 20}, {2, 2}, {9, 9}, {10, 10}},  // just last tile (communicate without transpose)
+    {{25, 25}, {5, 5}, {1, 1}, {1, 1}},    //
```

Comment:
```suggestion
    {{25, 25}, {5, 5}, {1, 1}, {1, 1}},
```
? Or comment missing?

Reply:
I added it to avoid clang-format merging ll 125 and 126.